README.md

```diff
@@ -0,0 +1,50 @@
+NOnion
+-------------------------------
+[![Build Status](https://github.com/nblockchain/NOnion/actions/workflows/CI.yml/badge.svg?branch=master&event=push)](https://github.com/nblockchain/NOnion/actions/workflows/CI.yml)
+
+_Unofficial_ work in progress TOR client library implemented in F#
```

Comment:
I would say `dotnet TOR-client library (implemented in F#)`, otherwise C# devs might think it's only consumable from F#